src/topbar/topbar.tsx

```diff
@@ -3,6 +3,12 @@ import * as React from 'react';
 import { Dropdown, Item } from './dropdown';
 import { NavigationItems } from './navigation-items';
 
+interface Icon {
```

Comment:
Maybe it makes sense to think about a abstraction of https://github.com/Webtrekk/Ampel-UI/blob/master/src/topbar/dropdown.tsx#L8 so that we are able to also use it for this use case. Would be a bit of a refactoring here.

Reply:
Ok, I refactored that interface and used it. Removed the Icon interface.